src/openaq/incremental_downloader_parallel.py

```
@@ -483,12 +490,14 @@ def download_country_all(self, country_code: str, country_id: int,
             batch_size = max(5, min(30, int(available_keys / max(avg_sensors * 1.5, 1))))  # 1.5 pages per sensor estimate
             print(f"  → Batch size: {batch_size} locations per batch")
             
-            i = start_index
+            i = 0  # Start from 0, since active_locations is already sliced
             while i < len(active_locations):
                 batch_end = min(i + batch_size, len(active_locations))
                 batch = [(idx, loc) for idx, loc in enumerate(active_locations[i:batch_end], start=i)]
```

Pylint (3.3.7)

[convention] Line too long (102/100)

(C0301)